contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java

```diff
@@ -18,25 +18,18 @@
  */
 package com.datatorrent.contrib.romesyndication;
 
-import com.datatorrent.contrib.romesyndication.RomeFeedEntry;
-import com.datatorrent.contrib.romesyndication.RomeStreamProvider;
-import com.datatorrent.contrib.romesyndication.RomeSyndicationOperator;
-import com.datatorrent.common.util.BaseOperator;
 import com.datatorrent.api.DAG;
 import com.datatorrent.api.DefaultInputPort;
 import com.datatorrent.api.LocalMode;
+import com.datatorrent.common.util.BaseOperator;
+import org.junit.*;
```

Comment:
Yes wonder why the idea settings didn't take..

On Tue, Nov 10, 2015 at 12:37 PM, David Yan <notifications@github.com>
wrote:

> In
> contrib/src/test/java/com/datatorrent/contrib/romesyndication/RomeSyndicationOperatorTest.java
> <https://github.com/apache/incubator-apex-malhar/pull/94#discussion_r44461242>
> :
>
> >  import com.datatorrent.api.DAG;
> >  import com.datatorrent.api.DefaultInputPort;
> >  import com.datatorrent.api.LocalMode;
> > +import com.datatorrent.common.util.BaseOperator;
> > +import org.junit.*;
>
> I remember it's no *
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-apex-malhar/pull/94/files#r44461242>.
>